modules/ROOT/pages/microsoft-azure-sb-connector.adoc

```diff
@@ -0,0 +1,190 @@
+|*Cron Expression*| (Optional) UNIX CRON expression that specifies when to trigger the receiver action. For example setting this field to `0 0 * * *`  triggers the receiver action at midnight (00:00) every day.
+|*Max messages to receive*| Maximum number of messages to receive during the scheduled operation.
+|===
+
+==== Subscription Listener
+
+Configure the Subscription Listener as follows:
+
+image::microsoft-azure-sb-topicreceiver.png[Topic Source]
+
+[%header%autowidth.spread]
+|===
+|Field |Description
+|*Topic* | Topic name to which to connect.
+|*Subscription*| Subscription to receive events. To receive events from the dead letter queue, specify `QueueName/$deadletterqueue`.
+|*Receive Mode*
+a|
+* MANUAL: User must manually acknowledge messages.
+* AUTOMATIC: Message is acknowledged from the queue and then deleted from the queue after it is received.
+If a message is not acknowledged, it times out and goes to the end of the queue.
+|*Server timeout (ms)*| Maximum duration within which the client keeps renewing the message lock if the processing of the message is not completed by the handler.
+|*Prefetch Count*| When `Prefetch` is enabled in any of the official Service Bus clients, the receiver acquires additional messages, up to the `PrefetchCount` limit. This can be more messages than what the application initially requested. Set this field to 0 to disable prefetching.
+|*Cron Expression*| (Optional) UNIX CRON expression that specifies when to trigger the receiver action. For example, setting this field to `0 0 * * *`  triggers the receiver action at midnight (00:00) every day.
+|*Max. Messages to Receive*| Maximum number of messages to receive during the scheduled operation.
+|===
+
+== Use Case: Studio
+
+=== Get Queue list
+
+image::microsoft-azure-sb-flow.png[General]
+
+. Create a new Mule application in Studio, and select an HTTP Listener as a source in the new flow.
+. Add a new HTTP Listener Configuration global element:
+.. Enter the following values for the `Host` and `Port` fields:
++
+[%header%autowidth.spread]
+|===
+|Parameter |Value
+|*Host* |0.0.0.0
+|*Port* |8081
+|===
++
+.. Click *Save*.
+. Assign the Global configuration to the HTTP Listener.
+. In the *Path* field for the HTTP Listener path, enter `/list`.
+. Drag and drop a new *Microsoft Azure Service Bus* component on the flow.
+. Configure the Microsoft Azure Service Bus connector global element with its environment values.
+. Select the `List queues` operation in the dropdown option.
+. Save and run the project as a Mule Application.
+
+== Use Case: XML
+
+[source,code]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+
+<mule xmlns:http="http://www.mulesoft.org/schema/mule/http" xmlns:microsoft-azure-service-bus="http://www.mulesoft.org/schema/mule/microsoft-azure-service-bus" xmlns:tracking="http://www.mulesoft.org/schema/mule/ee/tracking" xmlns="http://www.mulesoft.org/schema/mule/core" xmlns:doc="http://www.mulesoft.org/schema/mule/documentation"
```

Comment:
Should add line breaks so the code doesn't run off the right margin:
<mule xmlns:http="http://www.mulesoft.org/schema/mule/http" 
xmlns:microsoft-azure-service-bus="http://www.mulesoft.org/schema/mule/microsoft-azure-service-bus" xmlns:tracking="http://www.mulesoft.org/schema/mule/ee/tracking" 
xmlns="http://www.mulesoft.org/schema/mule/core" 
xmlns:doc="http://www.mulesoft.org/schema/mule/documentation"
xmlns:spring="http://www.springframework.org/schema/beans"
xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
xsi:schemaLocation="http://www.springframework.org/schema/beans 
http://www.springframework.org/schema/beans/spring-beans-current.xsd

...